dockerfiles/tue-env.Dockerfile

```diff
@@ -28,11 +32,15 @@ SHELL ["/bin/bash", "-c"]
 # Install commands used in our scripts and standard present on a clean ubuntu
 # installation and setup a user with sudo priviledges
 RUN apt-get update -qq && \
-    apt-get install -qq --assume-yes --no-install-recommends apt-transport-https apt-utils ca-certificates dbus dialog git keyboard-configuration lsb-release openssh-client sudo tzdata wget > /dev/null && \
-    # Add amigo user
+    echo "resolvconf resolvconf/linkify-resolvconf boolean false" | debconf-set-selections && \
+    apt-get install -qq --assume-yes --no-install-recommends apt-transport-https apt-utils bash-completion ca-certificates curl dbus debconf-utils dialog git keyboard-configuration lsb-release iproute2 iputils-ping net-tools openssh-client psmisc resolvconf sudo tzdata wget > /dev/null && \
```

Comment:
I don't like the `-qq`. As when `apt-get` fails with double quite. The logs are so short, it is almost impossible to find the problem.